installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc

```diff
@@ -46,7 +45,7 @@ The {product-title} installation program offers the following methods for deploy
 
 |Restricted network {ibm-z-name} with z/VM
 |
-|
+|&#10003;
 |xref:../../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[&#10003;]
```

Comment:
Out of scope for the PR, but just FYI that it's hard for me (mildly red-green colorblind) to tell that these checkmarks were clickable in the rendered version.